CLAUDE.md

```
@@ -0,0 +1,23 @@
+# Agent skills
+
+### Issue tracker
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)